homeassistant/components/wevolor/config_flow.py

```diff
@@ -0,0 +1,72 @@
+"""Config flow for Wevolor Control for Levolor Motorized Blinds integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pywevolor import Wevolor
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("host"): str,
+        vol.Required("support_tilt", default=False): bool,
+        vol.Required("channel_count", default=6): vol.All(int, vol.Range(min=1, max=6)),
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
+    """Validate the user input allows us to connect."""
+
+    wevolor = Wevolor(data["host"])
+    status = await hass.async_add_executor_job(wevolor.get_status)
+
+    if not status:
+        raise CannotConnect
+
+    return await status
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Wevolor Control for Levolor Motorized Blinds."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user", data_schema=STEP_USER_DATA_SCHEMA
+            )
+
+        errors = {}
+
+        try:
+            info = await validate_input(self.hass, user_input)
+        except CannotConnect:
+            errors["base"] = "cannot_connect"
+        except Exception as e:  # pylint: disable=broad-except
+            _LOGGER.exception(f"Unexpected exception: {e}")
```

Comment:
- Use percentage formatting for logging:
https://developers.home-assistant.io/docs/development_guidelines/#use-new-style-string-formatting

- `e` triggers C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name) 
```suggestion
        except Exception as err:  # pylint: disable=broad-except
            _LOGGER.exception("Unexpected exception: %s", err)
```

Reply:
thx!